src/EVM/Keccak.hs

```diff
@@ -50,8 +52,11 @@ combine lst = combine' lst []
       let xcomb = [ (x, y) | y <- xs] in
       combine' xs (xcomb:acc)
 
+maxW32 :: W256
+maxW32 = into (maxBound :: Word32)
+
 minProp :: Expr EWord -> Prop
-minProp k@(Keccak _) = PGT k (Lit 50)
+minProp k@(Keccak _) = PGT k (Lit maxW32)
```

Comment:
I think we only really need this to avoid potential collisions with value type storage slots? I agree that 50 might be a little small, but I would also probably prefer to keep our assumptions here as weak as we need to get usable results? What do you think about setting it to 256 instead (this seems to be safely enough for all real world solidity contracts?), or do you think that 2^32 is useful for other reasons?

Reply:
256 is a good idea, that was my initial thought, too!

Reply:
Ok, it's 256 all around now :)